### ISSUE TITLE
Propose a way to advertise our 3D Printing Service

### ISSUE THREAD

**Issue by MetaFight** [OxfordHackspace/HelperWork#7]
We are a hybrid Hackspace and Prototyping Service and when it comes to 3D printing we are currently pretty well equipped.  It would be nice to make this known to our potential clientèle.

How do we reach people?

If you have an idea please create and submit an advertising plan.

**Comment by NickKolpin** [comment]
@MetaFight, I don't think this is a quick task, but probably a discussion to have on the mailing list. Once we have a discussed a strategy then it we could create a list of tasks to act on it. I have some ideas for that, but I think discussing them on the mailing list would be the way forward. Do you want to start a thread about it?

**Comment by MetaFight** [comment]
I disagree that discussing this on the mailing list is a good idea. Doing that will just attract a bunch of opinions and very little actual action.

I think a better approach is having somebody draw up an advertising plan and then submit it for review/approval.  If it's good we implement it. If not, someone else gives it a go.
